File: Telerik Academy 2013-2014/02. C# Programming – Part II/08. Strings and Text Processing/StringsAndTextProcessing/ReverseString/ReverseString.cs
using System;
using System.Text;

class ReverseString
{
    static void Main()
    {
        Console.Write("The string to be reversed: ");
        string input = Console.ReadLine();
        StringBuilder result = new StringBuilder();

        for (int i = input.Length - 1; i >= 0; i--)
        {
            result.Append(input[i]);
        }

        Console.WriteLine("The reversed string: {0}", result.ToString());
    }
}

File: Telerik Academy 2013-2014/16. Android Mobile Applications/01. Java Exercises/JavaExercises/src/reverseString/ReverseString.java
package reverseString;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class ReverseString {
	public static void main(String[] args) {
		BufferedReader bufferedReader = new BufferedReader(
				new InputStreamReader(System.in));

		try {
			System.out.print("Enter string to be reversed: ");
			String input = bufferedReader.readLine();

			StringBuilder builder = new StringBuilder();

			for (int i = input.length() - 1; i >= 0; i--) {
				builder.append(input.charAt(i));
			}

			System.out.println("The reversed string is: " + builder.toString());
		} catch (IOException e) {
			e.printStackTrace();
		} finally {
			try {
				bufferedReader.close();
			} catch (IOException e) {
				e.printStackTrace();
			}
		}
	}
}